WikiWatershed/model-my-watershed
# Drop DRB-specific map layers from BigCZ

## Overview

This PR removes the DRB-specific layers from BigCZ. To accomplish this I added a `big_cz` key to each layer in layer settings, then read them into the client & added some if logic to check whether or not to show them contextually.

Connects #2395

### Demo

MMW:

![screen shot 2017-10-20 at 11 52 11 am](https://user-images.githubusercontent.com/4165523/31830122-48440ed4-b58d-11e7-8e72-f0dc839c7c78.png)

BigCZ:

![screen shot 2017-10-20 at 11 52 17 am](https://user-images.githubusercontent.com/4165523/31830129-4d5f02de-b58d-11e7-911b-be46f6cceced.png)

### Notes

I didn't touch the Observations tab, which also has some DRB layers.

## Testing Instructions
- get this branch, then `bundle`
- visit BigCZ & verify that only the layers mentioned in #2395 are included in their respective layer group
- visit MMW and verify that all the layers still appear
